tietokanta/src/main/resources/db/migration/R__Laskutusyhteenveto_mhu.sql

```diff
@@ -644,549 +654,646 @@ BEGIN
     IF indeksin_arvo > 0 THEN indeksi_puuttuu := FALSE; ELSE indeksi_puuttuu := TRUE; END IF;
     -- Aina syyskuu MHU urakoissa. Indeksi otetaan siis aina edellisen vuoden syyskuusta.
 
-    johto_ja_hallintakorvaus_toimenpideinstanssi_id := (SELECT tpi.id AS id
-                                                     FROM toimenpideinstanssi tpi
-                                                              JOIN toimenpide tpk3 ON tpk3.id = tpi.toimenpide
-                                                              JOIN toimenpide tpk2 ON tpk3.emo = tpk2.id,
-                                                          maksuera m
-                                                     WHERE tpi.urakka = ur
-                                                       AND m.toimenpideinstanssi = tpi.id
-                                                       AND tpk2.koodi = '23150'
-                                                     limit 1);
+    johto_ja_hallintakorvaus_toimenpideinstanssi_id := (
```

Comment:
Tässä on ollut alunperinkin ylimääräistä monimutkaisuutta. On turha hakea tietoja toimenpide.taso = 2 kautta. Käytetään suoraan tason 3 toimenpidekoodia (23151), eikä sotketa mukaan maksuerätaulua tarpeettomasti. Limit 1 myös pois. Jos rivejä palautuu liikaa, on hyväkin saada virhe.

SELECT tpi.id AS id
FROM toimenpideinstanssi tpi
         JOIN toimenpide tp ON tpi.toimenpide = tp.id
WHERE tpi.urakka = ur
  AND tp.koodi = '23151';